en/micro-integrator/docs/administer-and-observe/using-the-command-line-interface.md

```diff
@@ -379,32 +386,67 @@ mi version
          # To get details about a specific local entry
          mi localentry show  sampleLocalEntry
     ```
-- **secret**
-  ```
-   Usage:
-       mi secret [command] [arguments] [flags]
-
-   Available Commands:
-       init     Initialize the keystore information used for encryption
-       create   Create secrets based on preference
-     
-   Available Flags:
-       -f Use properties file to input multiple plaintext secrets
-         
-   Examples:
-       # To initialize keystore information
-       mi secret init
-       
-       # To encrypt secret and get output to console
-       mi secret create 
-       
-       # To encrypt secret and get output to file
-       mi secret create file
-       
-       # To encrypt secret and get output as a .yaml file
-       mi secret create k8
-       
-       # To bulk encrypt secrets defined in a properties file
-       mi secret create -f=</file_path>
+
+-   **user**
+    ```bash
+    Usage:
+       mi user [command] [flag]
+
+    Available Commands:
+       add              Add new user
+       remove           Remove an existing user
+       show             Get information about users
+
+    Flags:
+       -h, --help       Help for user
+       -p, --pattern    Specify a regex as a user name pattern
+       -r, --role       Specify a role to fetch a users        
+
+    Examples:
+       # To add a new user. This option is only available for admin users, therefore, set the `is-admin` flag to `true`
+       mi user add [new user-id] [password] [is-admin]
```

Comment:
This is changed with https://github.com/wso2/product-mi-tooling/pull/31

Reply:
Shall we apply these changes after merging this PR?